### ISSUE TITLE
"View > Show Tile Collision Shapes" doesn't apply to the tileset view

### ISSUE THREAD

**Issue by eevee** [mapeditor/tiled#3130]
This works when viewing a map, but not when viewing a tileset.  It'd be helpful for keeping track of collision shapes I've not yet added, as well as leftover shapes for tiles I've removed or changed.

**Comment by bjorn** [comment]
Definitely, thanks for bringing this up again! I left a comment at https://github.com/mapeditor/tiled/issues/1281#issuecomment-485009545 saying I wanted to do this, but I still did not get around to it.

**Comment by justdaft** [comment]
Is it possible to export the tile collision shapes that can be rendered on the map as an image.
